src/app/query/bitcoin/ordinals/ordinals.query.ts

```diff
@@ -146,24 +157,21 @@ export function useGetOrdinalsQuery() {
           .map(p => p.value);
 
         validatedResDataOrdApiInscriptions.forEach(data => {
```

Comment:
Why not use a map here, rather than `forEach`ing and pushing?

Reply:
Sure, will update